### PR TITLE
fix: auto create cli folder

### PR DESCRIPTION
**Pull request by foolishchow**
##### Checklist

- [x] commit message follows commit guidelines

##### Description of change

when run `egg-sequelize` , will auto create folder `migrations` and `app/model`

close https://github.com/eggjs/egg/issues/2090

### CODE REVIEW

File: lib/sequelizerc.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
 'use strict';
 const path = require('path');
+const fs = require('fs');
 
+const migrationsPath = path.resolve('migrations'),
+  modelsPath = path.resolve('app/model');
+if (!fs.existsSync(migrationsPath)) {
+  fs.mkdirSync(migrationsPath);
```
Comment (dead-horse) [inline lib/sequelizerc.js]: please use `mkdirp` module to ensure mkdir won't fail if parent directory, and also avoid check if migration path exists.

```js
mkdirp.sync(migrationPath);
```

File: lib/sequelizerc.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
 'use strict';
 const path = require('path');
+const fs = require('fs');
 
+const migrationsPath = path.resolve('migrations'),
```
Comment (atian25) [inline lib/sequelizerc.js]: one `const` one line.

File: lib/sequelizerc.js [hunk 3]
```diff
@@ -1,7 +1,19 @@
 'use strict';
 const path = require('path');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
 
+const migrationsPath = path.resolve('migrations');
+const modelsPath = path.resolve('app/model');
+
+if (!fs.existsSync(migrationsPath)) {
```
Comment (atian25) [inline lib/sequelizerc.js]: just use these, don't need `if`, cause `mkdirp` had do it for you.
```js
mkdirp.sync(migrationsPath);
mkdirp.sync(modelsPath);
```

**Comment by atian25** [comment]
ping @eggjs/core

**Comment by atian25** [comment]
@huacnlee 需要我这边发版本么？

**Comment by huacnlee** [comment]
可以的

**Comment by atian25** [comment]
3.1.2

**Pull request merged**